vmwareengine/cloud-client/create_policy.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START vmwareengine_create_policy]
+from google.api_core import operation
+from google.cloud import vmwareengine_v1
+
+
+def create_network_policy(
```

Comment:
If you'd like to use this sample in [Enable the internet access service in a region](https://cloud.google.com/vmware-engine/docs/networking/workload-internet-access#enable_the_internet_access_service_in_a_region) then we should add additional parameter to enable internet (similarly to what you've done in `update_network_policy` sample.

Reply:
Done.